src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -185,8 +200,8 @@ public void setpLossofHet(final double pLossofHet) {
      *                        of an individual sample to load (and exclude all others).
      * @return a Map of Sample name to Fingerprint
      */
-    public Map<String, Fingerprint> loadFingerprints(final Path fingerprintFile, final String specificSample) {
-        final VCFFileReader reader = new VCFFileReader(fingerprintFile, false);
+    public Map<String, Fingerprint> loadFingerprints(final Path fingerprintFile, final Path indexPath, final boolean forceIndex, final String specificSample) {
+        final VCFFileReader reader = getVCFReader(fingerprintFile, indexPath, forceIndex);
```

Comment:
what about 
```java
final VCFFileReader reader = new VCFFileReader(fingerprintFile, indexPath, forceIndex)
```

it looks like that constructor should handle a null indexPath correctly.

Reply:
This actually causes a bunch of tests to fail (apparently due to some null pointer exception). Although it seems like the method accepts `null` pointers, it looks like htsjdk handles that by assuming the index is next to the input file with extension `.tbi` which many of our test files don't have. I think it might be safer to leave it as is for now. This also allows for the custom PicardException to display when the user forces index file paths but they cannot be found.